packages/vue_frontend_rpc/local-dev/backend-mock.js

```diff
@@ -21,11 +23,14 @@ class BackendMock {
    * @param [opts.onFrontendReady] {Function}
    */
   constructor(opts) {
-    const actualOpts = defaults(opts, {
+    const initObject = {
       port: 8081,
-      save: ASYNC_NOOP,
-      onFrontendReady: ASYNC_NOOP,
-    });
+    };
+    for (const i in backendFuncNames) {
```

Comment:
very subjective: if we have lodash here we can use `zipObject` to avoid the loop

```javascript
const defaultFuncs = (new Array(backendFuncNames.length)).fill(ASYNC_NOOP)
const initObject = zipObject(backendFuncNames, [])
```

Reply:
OK cool :) I left it in a forEach loop for simplicity